src/database/DataTypes/Period.js

```diff
@@ -33,12 +33,21 @@ export class Period extends Realm.Object {
 
   requisitionsForOrderType(program, orderType) {
     return this.requisitions.filtered(
-      'program.id = $0 && orderType = $1',
+      'program.id = $0 && orderType = $1 && type == "request"',
       program.id,
       orderType.name
     ).length;
   }
 
+  customerRequisitionsForOrderTypeAndName(program, orderType, name) {
+    return this.requisitions.filtered(
+      'program.id = $0 && orderType = $1 && otherStoreName = $2',
+      program.id,
+      orderType.name,
+      name
+    ).length;
+  }
```

Comment:
Just finding requisitions which are specific to the customer. Additional filter otherwise, as we'll now have customer requisitions for the program and order type which shouldn't be counted toward the 'count' for supplier requisitions for the period/program/ordertype